pages/en/android-environment/deploying-apps.md

```diff
@@ -91,16 +95,47 @@ adb install [path to your APK]
 
 If you can't use the method described above and need to push your app from another device, you can connect the device to ADB using [USB](#connect-to-adb-over-usb) or a [network address](#connect-to-adb-over-a-network).
 
-To push your APK from another device into the Chromebook, you must start your Chrome OS in [developer mode](https://chromium.googlesource.com/chromiumos/docs/+/master/developer_mode.md) so that you can configure the Chromebook and push apps from the host machine. Follow this steps to get into [developer mode](/{{locale.code}}/productivity/experimental-features#developer-mode).
+### Connect to ADB over a network {: #adb-ip}
+
+1. Make sure you [enabled ADB debugging.](#enable-adb-debug)
+
+Get the IP address of your Chromebook:
+
+1.  Click the clock in the bottom-right area of the screen.
+1.  Click the gear icon.
+1.  Click the network type you are connected to (Wi-Fi or Mobile data) then the name of the network.
+1.  Take note of the IP Address.
+
+Connect to your Chromebook:
+
+1.  Return to your development machine and use ADB to connect to your Chromebook using its IP address:
+
+    ```bash
+    adb connect <ip_address>:22
```

Comment:
I don't believe adding the port is necessary here, but perhaps I just haven't run into the issue that this is guarding our users from.

Reply:
Good catch! Agreed, shouldn't need 22 anymore.